packages/ui/test/working-groups/modals/AddWorkerStakeModal.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { cryptoWaitReady } from '@polkadot/util-crypto'
+import { configure, fireEvent, render, screen } from '@testing-library/react'
+import React from 'react'
+
+import { Account } from '@/accounts/types'
+import { ApiContext } from '@/common/providers/api/context'
+import { ModalContext } from '@/common/providers/modal/context'
+import { AddWorkerStakeModal } from '@/working-groups/modals/AddStakeModal'
+
+import { getButton } from '../../_helpers/getButton'
+import { alice, bob } from '../../_mocks/keyring'
+import { MockKeyringProvider, MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+import {
+  stubApi,
+  stubDefaultBalances,
+  stubTransaction,
+  stubTransactionFailure,
+  stubTransactionPending,
+  stubTransactionSuccess,
+} from '../../_mocks/transactions'
+
+configure({ testIdAttribute: 'id' })
+
+const useMyAccounts: { hasAccounts: boolean; allAccounts: Account[] } = {
+  hasAccounts: true,
+  allAccounts: [],
+}
+
+jest.mock('@/accounts/hooks/useMyAccounts', () => {
+  return {
+    useMyAccounts: () => useMyAccounts,
+  }
+})
+
+describe('UI: AddStakeModal', () => {
+  setupMockServer()
```

Comment:
I think that we do not use query-node inside modal. If that's correct then you could remove `setupMockServer()` and query-node context from tests (should be a bit faster after that).